.github/workflows/ci.yml

```diff
@@ -730,7 +733,8 @@ jobs:
     runs-on: ubuntu-latest
     needs: [constraints]
     if: >
-      (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/v1-10-test' ) &&
+      (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/v1-10-test' ||
+      github.ref == 'refs/heads/v2-0-test') &&
```

Comment:
Same here. We do not need separate constraints for 2.0 until we continue rebasing.